client/src/app/globals.css

```
@@ -349,3 +349,53 @@ template {
 [hidden] {
   display: none;
 }
+
+/* Primitives
+   ========================================================================== */
+:root {
+  --default-font-family: "SUIT-Regular", sans-serif;
+  --default-font-style: normal;
+  --default-line-height: normal;
+  --spacing05: 4px;
+  --spacing1: 8px;
+  --spacing2: 16px;
+  --spacing3: 24px;
+  --spacing4: 32px;
+  --spacing5: 40px;
+  --spacing6: 48px;
+  --spacing7: 56px;
+  --spacing8: 64px;
+  --spacing9: 72px;
+  --spacing10: 80px;
+  --spacing11: 88px;
+  --spacing12: 96px;
+  --spacing13: 104px;
+  --radius-sm: 4px;
```

```suggestion
  --radius-sm: var(--primitives_spacing-05);
```

얘네들은 토큰이니 뭐 `--token_` prefix를 붙일 수도 있지만, 어차피 primitives와 tokens 밖에 없고 둘 중 token을 사용할 일이 훨씬 많은 것 같아서 그냥 prefix 없는 애들을 token으로 생각하는 게 경제적이지 않을까 싶으. 다만 토큰 값은 수치를 직접 입력하기 보다는 primitives를 가져오는 게 직관적일듯?